src/test/integration/addons-test.js

```diff
@@ -188,6 +188,9 @@ describe('addons', function() {
     expect(list[0]).toHaveProperty('description');
     expect(list[0]).toHaveProperty('version');
     expect(list[0]).toHaveProperty('url');
+    expect(list[0]).toHaveProperty('api');
```

Comment:
Shouldn't this be moziot.api ?

Reply:
No, that's the `package.json` format. The addons list format is different.

Reply:
ok - this is just failing then because the other change wasn't merged yet